### DISCUSSION TITLE
can't install airflow 2.3.0 with "mysql" extra, got some duplicate key

### DISCUSSION THREAD

**Discussion by chusri** [apache/airflow#23553] [General] [answered]
I created a Conda env with python 3.7 and install airflow by following step

1. install airflow with MySQL extra.

`pip install 'apache-airflow[mysql]==2.3.0' \ --constraint "https://raw.githubusercontent.com/apache/airflow/constraints-2.3.0/constraints-3.7.txt"`

2. initialized DB by running `airflow db init`
3. add admin user by running `airflow users create --role Admin --username admin --firstname fadmin --lastname ladmin --email admin@example.org`
4. I start airflow by using. `airflow webserver -p 8080` so the error happens.

<img width="1409" alt="Screen Shot 2022-05-06 at 8 16 55 AM" src="https://user-images.githubusercontent.com/2667287/167235883-edd0809d-9b59-456e-a761-684fedc8b62f.png">

<img width="432" alt="Screen Shot 2022-05-06 at 8 17 00 AM" src="https://user-images.githubusercontent.com/2667287/167235887-ed6c97e0-ead3-48dc-8d75-07a024324d5c.png">

**Comment by potiuk** [comment, marked as answer]
There is an issue for that https://github.com/apache/airflow/issues/23512 - it's aimed to be fixed in 2.3.1
